### PR TITLE
Add my new Raycast extension: Dropstab Crypto Toolbox

### PR DESCRIPTION
**Pull request by Motion-Up**
### Search
Dropstab Crypto Toolbox displays the price, market capitalization, and rank of any cryptocurrency, and also allows quick access to the project's page on Dropstab.com or its X (Twitter) profile.

DropsTab

### Wallets
Dropstab Crypto Toolbox allows you to instantly find and view any crypto address in the blockchain explorer.
DropsTab

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Motion-Up** [comment]
Good afternoon,

I am a representative of the DropsTab team. We would like to add an extension from our organization, but we are unable to enable the "Allow edits from maintainers" option because the PR is coming from the organization. Is there a way to add our extension from the organization, or do we need to fork it to a personal account and then create the PR?

**Comment by pernielsentikaer** [comment]
If you can compress all the images and make the corrections I tell you, then we can proceed with that 🙂

**Comment by Motion-Up** [comment]
> If you can compress all the images and make the corrections I tell you, then we can proceed with that 🙂

I have compressed the images. What corrections do I need to make?

**Comment by pernielsentikaer** [comment]
Can you add icons to all actions in the panel?

dropstab 2024-08-22 at 06 55 18

**Comment by Motion-Up** [comment]
I have added icons to all actions in the panel and removed the unnecessary code.

File: extensions/dropstab_raycast/src/search-crypto.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      icon={{ source: searchResult.icon || "", mask: Image.Mask.RoundedRectangle }}
+      title={`${searchResult.symbol}`} // Изменено для отображения rank перед символом
+      subtitle={searchResult.name}
+      accessories={[{ text: `${searchResult.rank} | ${searchResult.price} | ${searchResult.marketCap}` }]}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.OpenInBrowser title="Open in Dropstab" url={searchResult.url} icon="dropstab-logo.png" />
+            {isFavoriteForCoins(searchResult) ? (
+              <ActionPanel.Item
```
Comment (pernielsentikaer) [inline extensions/dropstab_raycast/src/search-crypto.tsx:106]:
```suggestion
              <Action
```
`<ActionPanel.Item` is deprecated

**Review by pernielsentikaer — COMMENTED** [review]
Could we move the extension to a folder just called `dropstab` instead?

File: extensions/dropstab_raycast/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [Unreleased]
+### Added
+- Initial release of the `dropstab_raycast` extension.
```
Comment (pernielsentikaer) [inline extensions/dropstab_raycast/CHANGELOG.md:5]: ## [Initial release] - 2024-08-22
- Initial release of the `dropstab_raycast` extension.

**Comment by Motion-Up** [comment]
I have renamed the folder to dropstab, checked the imports in the project, and updated the CHANGELOG.md file.

**Comment by pernielsentikaer** [comment]
I think it would be nice with an [argument](https://developers.raycast.com/information/lifecycle/arguments) for Wallet Address, what do you think? Then it would be even faster

Raycast22082024-TfwPBdGE png@2x

**Comment by Motion-Up** [comment]
Great idea! I have added an argument for Wallet Address. Thank you!

**Comment by Motion-Up** [comment]
I have fixed the Empty State Flicker issue. The other fix will be included in the next major update.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by Motion-Up** [comment]
Thank you for acknowledging my contribution🤝 I kindly request you to double-check the profile @Motion-Up, as it is registered on Raycast.

**Pull request merged**